### PR TITLE
Add io.github.cxong.cdogs-sdl

### PR DESCRIPTION
**Pull request by librebob**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/cxong/cdogs-sdl/issues/581
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* It wouldn't be valuable for upstream to disable warnings as errors.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by librebob** [comment]
bot, build io.github.cxong.cdogs-sdl

File: io.github.cxong.cdogs-sdl.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                "type": "archive",
+                "url": "https://files.pythonhosted.org/packages/3d/64/a3b379cb9c7827ad33c67dcda4c4ad117bdef1b7d68b22a05c963cf4727d/protobuf-3.17.3.tar.gz",
+                "sha256": "72804ea5eaa9c22a090d2803813e280fb273b62d5ae497aaf3553d141c4fdd7b"
+            }
+        ]
+    },
+      {
+          "name": "cdogs-sdl",
+          "buildsystem": "simple",
+          "build-commands": [
```
Comment (hfiguiere) [inline io.github.cxong.cdogs-sdl.json:51]: You could split between a `shell` source to run before build and a `post-install`, and use `cmake-ninja`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by librebob** [comment]
bot, build io.github.cxong.cdogs-sdl

**Pull request closed without merging**